python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -756,9 +756,9 @@ def getGainFromFlatPair(self, im1Area, im2Area, imStatsCtrl, mu1, mu2,
         if correctionType not in ['NONE', 'SIMPLE', 'FULL']:
             raise RuntimeError("Unknown correction type: %s" % correctionType)
 
-        if correctionType != 'NONE' and readNoise is None:
+        if correctionType != 'NONE' and not np.isfinite(readNoise):
```

Comment:
Why ``np.isfinite`` instead of ``np.isnan``? Just curious, I guess that, for this case, they work the same?

Reply:
No particular reason.  `isfinite` catches other values that aren't plausible as well, but there are other impossible values (negative values).  I'm going to leave it for now.